cpp/daal/include/algorithms/covariance/covariance_types.h

```diff
@@ -380,7 +388,8 @@ class DAAL_EXPORT DistributedInput<step1Local> : public Input
 {
 public:
     DistributedInput() : Input() {}
-    DistributedInput(const DistributedInput & other) : Input(other) {}
+    DistributedInput(const DistributedInput & other)             = default;
+    DistributedInput & operator=(const DistributedInput & other) = default;
```

Comment:
As for this class I can't find a respective cpp with implementation, so I'm not sure if we can move the implementation of this functions from here. I don't understand the purpose of DistributedInput<step1Local> maybe it should removed in the future